lib/src/view/broadcast/broadcast_screen.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:chessground/chessground.dart';
+import 'package:fast_immutable_collections/fast_immutable_collections.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:lichess_mobile/src/model/broadcast/broadcast.dart';
+import 'package:lichess_mobile/src/model/broadcast/broadcast_providers.dart';
+import 'package:lichess_mobile/src/utils/chessground_compat.dart';
+import 'package:lichess_mobile/src/widgets/board_thumbnail.dart';
+import 'package:lichess_mobile/src/widgets/layout_board.dart';
+import 'package:lichess_mobile/src/widgets/platform.dart';
+import 'package:lichess_mobile/src/widgets/shimmer.dart';
+import 'package:lichess_mobile/src/widgets/user_full_name.dart';
+
+class BroadcastScreen extends ConsumerWidget {
+  final String roundId;
+
+  const BroadcastScreen({super.key, required this.roundId});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return PlatformWidget(
+      androidBuilder: _androidBuilder,
+      iosBuilder: _iosBuilder,
+    );
+  }
+
+  Widget _androidBuilder(
+    BuildContext context,
+  ) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Broadcast'),
```

Comment:
We should translate this. I'll write the documentation that explain how to add a new i18n module, so you can do it yourself and verify my doc is good ;)

Reply:
Here is the doc: https://github.com/lichess-org/mobile/blob/main/docs/internationalisation.md

Reply:
Thanks for the documentation. The only issue I had is that I didn't have nodejs nor the modules used by the script so I had to installed them. Maybe you could add in the documentation the version of nodejs that this script works with.